README.md

```diff
@@ -1,15 +1,15 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
+capstone1-concert-website
 
 > Description the project.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML CSS and javascript
+- Sass and bootstrap
 
 ## Live Demo (if available)
```

Comment:
- [ ] Kindly add the link to your live demo project 👍🏽

Reply:
- [ ] Kindly make sure your page has been deployed and accessible on the internet.
`hint: You can use GitHub pages to do that`